src/test/java/io/pravega/connectors/flink/FlinkPravegaTableITCase.java

```diff
@@ -358,15 +374,14 @@ static boolean compare(List<Row> results, List<Row> expectedResults) {
                 { "Nina",  "2018-08-02 09:40:00.0", "1" },
                 { "Peter", "2018-08-02 09:45:00.0", "1" },
                 { "Tony",  "2018-08-02 09:45:00.0", "1" },
-                { "Tony",  "2018-08-02 10:45:00.0", "1" }
-        };
+                //{ "Tony",  "2018-08-02 10:45:00.0", "1" },
```

Comment:
The watermark is not passing the `10:45`, so the window will not be fired and compute the count, thus we don't have this line for output. Please put the annotation here.